blockly-core/core/utils/xml.js

```diff
@@ -17,6 +17,12 @@
  * limitations under the License.
  */
 
+/* global Blockly, goog */
+
+// Ignore "Expected a conditional expression and instead saw an
+// assignment" warnings.
+/* jshint -W084 */
```

Comment:
I think as we write new code in blockly-core, we tend to move it more towards our style.

If there were already a whole lot of these in the file, does that mean it was previously failing linting?